### PR TITLE
add Youtubeast integration

### PR DESCRIPTION
**Pull request by Hanzalah-Adalan**
huhuu.. please review.. and pardon the copy paste part 😂

### CODE REVIEW

File: core/lib/youtube/master.js [hunk 1]
```diff
@@ -7,6 +8,7 @@ export default {
     icon: "https://arlo.site/projects/UWPCompanion/icons/platforms/YouTube.png",
     baseUrlMatch: YTParser.isYoutube,
     clients: {
-        myTube
+        myTube,
+        Youtubeast
```
Comment (aelisya) [inline core/lib/youtube/master.js]: Review

File: core/lib/youtube/youtubeast.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        pauseVideo(tabId);
+    }
+}
+
+export default {
+    name: "Youtubeast",
+    parseUrl: getProtocolFromUrl,
+    postLaunch: postLaunch,
+    config: {
+        logo: "https://res.cloudinary.com/dfbht6vj9/image/upload/v1562960997/Logo_225_rbe0nf.png",
```
Comment (Arlodotexe) [inline core/lib/youtube/youtubeast.js]: Resource change requested (See review comment)

File: core/lib/youtube/youtubeast.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        icon: "https://res.cloudinary.com/dfbht6vj9/image/upload/v1562960997/icon_48_faqsgu.png",
```
Comment (Arlodotexe) [inline core/lib/youtube/youtubeast.js]: Resource change requsted (See review comment)

Comment (Hanzalah-Adalan) [reply core/lib/youtube/youtubeast.js]: Oh I see, alright I'll submit another PR as soon as I fix the breaking bug that I just discovered this morning.. 🤦‍♂️

**Pull request closed without merging**